.qawolf/tests/sendMessage.test.js

```diff
@@ -13,7 +13,12 @@ describe('sendMessage', () => {
   it('can type into "message" input', async () => {
     await browser.type(selectors[1], "OK Google");
     await browser.click(selectors[3]);
-    const hasAddedMessage = browser.find({css: ".messages-wrapper"});
-    expect(hasAddedMessage.innerText).toContain("OK Google")
+
+    const hasMessage = await browser.hasText("OK Google");
+    expect(hasMessage).toEqual(true);
+
+    const messagesElement = await browser.find({css: ".messages-wrapper"});
```

Comment:
These are two different ways to write the assertion. `find` is a promise, so you need to `await` it. It returns an [`ElementHandle`](https://github.com/puppeteer/puppeteer/blob/v2.0.0/docs/api.md#class-elementhandle) so you need to call [`.evaluate`](https://github.com/puppeteer/puppeteer/blob/v2.0.0/docs/api.md#elementhandleevaluatepagefunction-args) to get the innerText.

Reply:
Thanks for the PR !!! ✨　It's working as expected.
I think I've learned a lot. 

Reply:
Awesome! :)